src/Hooks/FieldStorageConfigEditFormAlter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\file_encrypt\Hooks;
+
+use Drupal\Core\Field\FieldTypePluginManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\encrypt\EncryptionProfileManagerInterface;
+use Drupal\field\FieldStorageConfigInterface;
+use Drupal\file\Plugin\Field\FieldType\FileItem;
+
+class FieldStorageConfigEditFormAlter {
+
+  use StringTranslationTrait;
```

Comment:
Does this go here or outside of the class? I rarely see use statements inside of classes.